zwave_js_server/model/node.py

```diff
@@ -511,6 +512,30 @@ async def async_ping(self) -> bool:
         )
         return cast(bool, data.get("responded", False))
 
+    async def async_has_security_class(self, security_class: SecurityClass) -> bool:
+        """Return whether node has the given security class."""
+        data = (
+            await self.async_send_command(
+                "has_security_class",
+                securityClass=security_class,
+                require_schema=8,
+                wait_for_result=True,
+            )
+            or {}
+        )
+        return cast(bool, data["hasSecurityClass"])
```

Comment:
There will be a KeyError if the default empty dict is returned above.

Reply:
The reason for the empty dict here is to get past mypy. Unless the command fails, we will always get `hasSecurityClass`

Reply:
If we know that `async_send_command` will return a non empty dict we should just `assert data`.